### ISSUE TITLE
Overwriting logits from all conversations

### ISSUE THREAD

**Issue by hvgazula** [hassonlab/247-pickling#113]
I may be overwriting logits from all conversations into one file. Check this.
